polling_stations/apps/data_importers/management/commands/import_reigate_and_banstead.py

```diff
@@ -4,29 +4,39 @@
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "REI"
     addresses_name = (
-        "2023-05-04/2023-04-05T12:04:27.040829/Democracy_Club__04May2023.tsv"
+        "2024-05-02/2024-03-05T11:26:20.067521/Democracy_Club__02May2024.tsv"
     )
     stations_name = (
-        "2023-05-04/2023-04-05T12:04:27.040829/Democracy_Club__04May2023.tsv"
+        "2024-05-02/2024-03-05T11:26:20.067521/Democracy_Club__02May2024.tsv"
     )
-    elections = ["2023-05-04"]
+    elections = ["2024-05-02"]
     csv_delimiter = "\t"
 
-    def station_record_to_dict(self, record):
-        # Woodmansterne Village Hall, Carshalton Road, Woodmansterne, Banstead, Surrey
-        if record.polling_place_id == "4916":
-            record = record._replace(
-                polling_place_easting="527572", polling_place_northing="160117"
-            )
-
-        return super().station_record_to_dict(record)
-
     def address_record_to_dict(self, record):
         uprn = record.property_urn.lstrip("0")
 
         if uprn in [
-            "68181592",  # 2A PARKHURST ROAD, HORLEY
+            "68134095",  # CHIDWELL FARMING, NICOLA FARM, 37 WWOODMANSTERNE LANE, BANSTEAD
+            "68137043",  # 170 DOVERS GREEN ROAD, REIGATE
+            "68137147",  # 168 DOVERS GREEN ROAD, REIGATE
+            "68183366",  # MYRTLE COTTAGE, HORLEY LODGE LANE, REDHILL
+            "68115368",  # 1 DEAN LANE, MERSTHAM, REDHILL
+        ]:
+            return None
+
+        if record.addressline6 in [
+            # splits
+            "RH6 8DU",
         ]:
             return None
 
         return super().address_record_to_dict(record)
+
+    def station_record_to_dict(self, record):
+        # more accurate point for: Woodmansterne Village Hall, Carshalton Road, Woodmansterne, Banstead, Surrey
+        if record.polling_place_id == "5001":
+            record = record._replace(
+                polling_place_easting="527572", polling_place_northing="160117"
+            )
+
+        return super().station_record_to_dict(record)
```

Comment:
Not a huge problem, but reordering the methods makes it a little more confusing to review. This one is easy, but if there are several changes it gets confusing.